crossfire/__init__.py

```diff
@@ -43,17 +43,18 @@ def occurrences(
     )
 
 
-def _flatten_dict(data, parent_key, sep="_"):
```

Comment:
Removed the `_flatten_dict`, as `flatten` function, for now will run only on `dict`.